stories/components/modal.stories.tsx

```diff
@@ -255,12 +238,21 @@ export function DynamicallySizedChildren() {
       <Button intent="warning" onClick={open}>
         Open editor modal
       </Button>
-      <Modal width={700} height={500} isOpen={isOpen} onRequestClose={close}>
-        <Modal.Header>Test OCL editor in modal</Modal.Header>
-        <Modal.Body>
+      <Dialog
+        title="Test OCL editor in modal"
+        isOpen={isOpen}
+        onClose={close}
+        style={{ width: 700, maxHeight: 500 }}
```

Comment:
is the `maxHeight` property really needed here?

Reply:
No, I just used the same old stories values.